packages/ui/src/theme/createTheme.ts

```diff
@@ -63,6 +63,38 @@ function setupTokens(obj: any, path = []) {
   return tokens;
 }
 
+/**
+ * This function takes in an array of WebDesignTokens and filters out the duplicate WebDesignTokens
+ * that are defined in the DUPLICATE_STATE_TOKENS list. This is designed to remove the duplicate tokens
+ * that were created due to inconsistent naming.  This should be removed when the tokens are removed in the
+ * next major version release.
+ * @param tokens
+ * @returns WebDesignTokens[]
+ */
+function removeDuplicateStateTokens(
+  tokens: WebDesignToken[]
+): WebDesignToken[] {
+  let tokenList = tokens;
```

Comment:
If you do `tokenList = tokens` hoping to keep the values in `tokens`, it doesn't work. Is that L77 for?

Try this
```
let a = [1,2,3];
let b = a;
b[0]++;
console.log(a); // [2,2,3]
```

Reply:
To keep `tokens` unchanged, you can do `let tokenList = [...tokens]`;

Reply:
tokenList has been removed because it is no longer needed as we are just doing a single pass now